RFC-0039-cuda-support.md

```diff
@@ -0,0 +1,196 @@
+
+# [CUDA version support]
+
+**Authors:**
+* @atalman @malfet
+
+## **Motivation**
+
+The proposal is to provide two main benefits
+
+- Aligned decision policy provides transparent decision making;
+- Incorporate decision points in the early release process to stagger CUDA migration from the RC release integration work.
+
+## **Proposed Policy and Process**
```

Comment:
Under this proposal, would any of our decisions to upgrade Cuda in the past year have changed?

For example, in the past year, which releases would we have chosen to _not_ upgraded Cuda?

Reply:
Yes, this is a history of decisions we took for past year https://github.com/pytorch/rfcs/pull/74#discussion_r2096005474 

We supported CUDA: 11.8, 12.1, 12.4, 12.6 and 12.8 .